cdap-integration-test/src/main/java/io/cdap/cdap/test/IntegrationTestBase.java

```diff
@@ -366,21 +365,24 @@ protected DatasetClient getDatasetClient() {
 
   protected ApplicationManager deployApplication(NamespaceId namespace,
                                                  Class<? extends Application> applicationClz,
-                                                 File...bundleEmbeddedJars) throws IOException, AccessException {
+                                                 File...bundleEmbeddedJars) throws Exception {
+    checkSystemServices();
     return getTestManager().deployApplication(namespace, applicationClz, bundleEmbeddedJars);
   }
 
   protected ApplicationManager deployApplication(Class<? extends Application> applicationClz)
-    throws IOException, AccessException {
+    throws Exception {
```

Comment:
why do we need to change this and not catch specific exceptions?

Reply:
With this change, 2 other exceptions need to be added, it becomes a long list and it is repeating in the other function which invokes this. Right way is to wrap it with user defined exceptions. Not sure if that is needed for integration tests.